src/main/resources/db/migration/common/V1_107__insert_transcription_types_reference_data.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (1, 'Sentencing remarks');
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (4, 'Argument and submission of ruling');
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (5, 'Court Log');
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (6, 'Mitigation');
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (7, 'Proceedings after verdict');
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (8, 'Prosecution opening of facts');
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (9, 'Specified Times');
+INSERT INTO darts.transcription_type (trt_id, description) VALUES (10, 'Other');
```

Comment:
Do we need to restart the sequence `trt_seq` here - since we have one?

Reply:
done, and added a restart to courthouse also.